crates/issue/src/lib.rs

```diff
@@ -348,6 +350,7 @@ impl<T: Config> Pallet<T> {
             amount: amount_user.amount(),
             fee: fee.amount(),
             griefing_collateral: griefing_collateral.amount(),
+            griefing_currency: griefing_collateral.currency(),
```

Comment:
Why not just change the type of `griefing_collateral` from `Balance` to `Amount`, to include `griefing_currency`?

Reply:
I'm assuming that it messes up storage byte alignment